main.cpp

```diff
@@ -0,0 +1,419 @@
+#include <SFML/Graphics.hpp>
+#include <SFML/Network.hpp>
+#include <iostream>
+#include <windows.h>
+#include <map>
+#include "TextureHolder.h"
+#include "Square.h"
+#include "Piece.h"
+
+bool move(Square* slctd, Square* trgt, sf::UdpSocket& sckt, sf::IpAddress& IP, unsigned short& PORT, sf::Packet& pck);
+
+int main()
+{	
+	//kod debugowania = 1337
+	bool turn=false, gamestarted=false, iswhite = true;
+	sf::Packet pack;
+	sf::UdpSocket socket;
+	sf::IpAddress serveradress;
+	unsigned short serverport;
+	int serverconnectionstatus = 1, id, ctx, cty;
+	float tx, ty; //koordy targetu do update
+
+
+
+
+	std::cout << "Server IP: ";
+	std::cin >> serveradress;
+	std::cout << "Server Port: ";
+	std::cin >> serverport;
```

Comment:
Nie lepiej to brać jako argument przy uruchomieniu programu?
```cpp
int main(int argc, char **argv) { ... }
```
wtedy `argc` = liczba argumentów, a `argv` to ich lista
[Artykuł o tym](https://www.geeksforgeeks.org/command-line-arguments-in-c-cpp/)